testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/TestRun.java

```diff
@@ -121,6 +122,9 @@ public Integer call()
         @Option(names = "--attach", description = "attach to an existing environment")
         public boolean attach;
 
+        @Option(names = "--debug-suspend-tests", description = "Wait for client to connect in debug mode. Product Tests process only.")
```

Comment:
Maybe it should be default behavior for tests container?

Reply:
no, it shouldn't

Reply:
Or we could have debug modes like: `--debug=suspend_tests`?